docs/readthedocs/source/conf.py

```diff
@@ -244,3 +245,6 @@ def setup(app):
         'enable_auto_doc_ref': True,
     }, True)
     app.add_transform(AutoStructify)
+
+# disable notebook execution
+nbsphinx_execute = 'never'
```

Comment:
When clearing all outputs of the ipynb files, nbsphinx seems to execute the notebook by default, which will cause errors like this: [add-nano-tutorials-to-docs](https://readthedocs.org/projects/yuwentestdocs/builds/17569373/).

According to the nbsphix documentation (https://nbsphinx.readthedocs.io/en/0.8.9/never-execute.html), execution is disabled.